source/android/adaptivecards/src/androidTest/java/io/adaptivecards/objectmodel/FeatureRegistrationRenderTest.java

```diff
@@ -0,0 +1,64 @@
+package io.adaptivecards.objectmodel;
+
+import android.content.Context;
+import android.support.test.InstrumentationRegistry;
+import android.test.mock.MockContext;
+import android.widget.LinearLayout;
+
+import junit.framework.Assert;
+
+import org.junit.Test;
+
+import io.adaptivecards.renderer.AdaptiveCardRenderer;
+import io.adaptivecards.renderer.RenderedAdaptiveCard;
+import io.adaptivecards.renderer.registration.CardRendererRegistration;
+
+public class FeatureRegistrationRenderTest
+{
+
+    static {
+        System.loadLibrary("adaptivecards-native-lib");
+    }
+
+    @Test
+    public void TestDefaultRendering() throws Exception
```

Comment:
nit: maybe edit the title of the test to be more descriptive to teh problem - ie TestDefaultRenderingWithNoFeatureRegistrationSpecified or something